packages/node_modules/@ciscospark/react-component-activity-share-files/src/index.js

```diff
@@ -101,7 +106,14 @@ export function ActivityShareFiles(props) {
   return (
     <ActivityItemBase {...props}>
       <div className={classNames('ciscospark-activity-share-list', styles.shareList)}>
-        {renderAdaptiveCard && <AdaptiveCard cards={cards} displayName={displayName} />}
+        {renderAdaptiveCard && <AdaptiveCard
+          cards={cards}
+          displayName={displayName}
+          items={files}
+          verb={verb}
```

Comment:
Is this component the one [defined in this file](https://github.com/webex/react-widgets/blob/5b1a635ef2b1e5eb25e53cf1b45ef0330dcb3df6/packages/node_modules/%40ciscospark/react-component-adaptive-card/src/index.js)? Because looking at the [props defined](https://github.com/webex/react-widgets/blob/5b1a635ef2b1e5eb25e53cf1b45ef0330dcb3df6/packages/node_modules/%40ciscospark/react-component-adaptive-card/src/index.js#L102) they don't seem to be matching with what is being called here

Reply:
Refer:https://github.com/webex/react-widgets/pull/1034#discussion_r356935122